source/operations/concepts/architecture.rst

```diff
@@ -0,0 +1,189 @@
+
+   .. figure:: /images/architecture/architecture-multiple-clients.svg
+      :figwidth: 100%
+      :alt: Diagram of multiple S3-compatible clients using SDKs to connect to MinIO
+
+      Clients using a variety of S3-compatible SDKs can perform operations against the same MinIO deployment.
+
+   MinIO uses a strict implementation of the S3 API, including requiring clients to sign all operations using AWS :s3-api:`Signature V4 <sig-v4-authenticating-requests.html>` or the legacy Signature V2.
+   AWS signature calculation uses the client-provided headers, such that any modification to those headers by load balancers, proxies, security programs, or other components can result in signature mismatch errors.
+   Ensure any such intermediate components support pass-through of unaltered headers from client to server.
+
+   The complexity of signature calculation typically makes interfacing via CURL or similar REST clients difficult or impractical. 
+   MinIO recommends using S3-compatible drivers which perform the signature calculation automatically as part of operations.
+
+Replicated MinIO Deployments
+----------------------------
+
+MinIO :ref:`site replication <minio-site-replication-overview>` provides support synchronizing distinct independent deployments.
+   You can deploy peer sites in different racks, datacenters, or geographic regions to support functions like :abbr:`BC/DR (Business Continuity / Disaster Recovery)` or geo-local read/write performance in a globally distributed MinIO object store.
+
+   .. figure:: /images/architecture/architecture-multi-site.svg
+      :figwidth: 100%
+      :alt: Diagram of a multi-site deployment with three MinIO peer site
+
+      A MinIO multi-site deployment with three peers.
+      Write operations on one peer replicate to all other peers in the configuration automatically.
+
+Each peer site consists of an independent set of MinIO hosts, ideally having matching pool configurations.
+   The architecture of each peer site should closely match to ensure consistent performance and behavior between sites.
+   All peer sites must use the same primary identity provider, and during initial configuration only one peer site can have any data.
+
+   .. figure:: /images/architecture/architecture-multi-site-setup.svg
+      :figwidth: 100%
+      :alt: Diagram of a multi-site deployment during initial setup
+
+      The initial setup of a MinIO multi-site deployment.
+      The first peer site replicates all required information to other peers in the configuration.
+      Adding new peers uses the same sequence for synchronizing data.
+
+Replication performance primarily depends on the network latency between each peer site.
+   With geographically distributed peer sites, high latency between sites can result in significant replication lag.
+   This can compound with workloads that are near or at the deployment's overall performance capacity, as the replication process itself requires 
```

Comment:
Need a `.` at the end.

I'm not sure `as the replication process itself requires` is a complete thought. Maybe just drop that?